src/main/java/com/netgrif/application/engine/petrinet/domain/version/Version.java

```diff
@@ -11,7 +11,7 @@
 @AllArgsConstructor
 public class Version {
 
-    public static final String NEWEST = "^";
+    public static final String NEWEST = "latest";
```

Comment:
As process versions follow semantic versioning format, it's better to rename the constant `LATEST`.